### PR TITLE
Setup firebase release github action

### PR DESCRIPTION
**Pull request by StianHaugland1**

### CODE REVIEW

File: .github/workflows/distribution.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+name: Distribution
+on:
+  push:
+    branches:
+      - main
+
```
Comment (konstantpapp) [inline .github/workflows/distribution.yml:6]: maybe add on 'workflow_dispatch: ' in addition to push trigger so we can run the action whenever :)

File: android/app/build.gradle [hunk 2]
```diff
@@ -50,17 +50,26 @@ android {
...
-        release {
-            // TODO: Add your own signing config for the release build.
-            // Signing with the debug keys for now, so `flutter run --release` works.
+    
+    signingConfigs {
+       release {
+           keyAlias System.getenv('KEY_ALIAS')
+           keyPassword System.getenv('STORE_PASSWORD')
+           storeFile file(System.getenv('STORE_FILE'))
+           storePassword System.getenv('STORE_PASSWORD')
```
Comment (konstantpapp) [inline android/app/build.gradle:62]: Just doublechecking, is the env var for keyPassword correct?

**Review by konstantpapp — APPROVED** [review]
♾️ 🚀

File: android/app/build.gradle (hunk 2 above)
Comment (StianHaugland1) [reply android/app/build.gradle:62]: Yes

**Pull request merged**